tools/android/build_aar.py

```diff
@@ -55,6 +55,7 @@ def build_for_abi(
 def build_aar(
     output_dir: Path,
     config: str,
+    build_target: str,
```

Comment:
nit: confusing to have a file and function called `build_aar` that is building an AAR or a shared library.